config/config.example.yml

```diff
@@ -135,7 +131,7 @@ sync_permissions:
       - "nginx_workspace : write <-> thredds_workspace : write"
   weaver_outputs:
     services:
-      weaver:
```

Comment:
Just to be sure you don't modify too many things for nothing... `weaver` and `weaver-wps-outputs` are effectively 2 different services. The first is a `ServiceAPI` protecting a `weaver` instance itself, and the second is the nginx directory were files are downloaded from, that will also be protected with another `ServiceAPI` instance.